README.md

```diff
@@ -1,74 +1,87 @@
-# GitOpener
+# GitOpener 🚪🔓
 
 [![Open in GitPod](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/pbclife/gitopener.vercel.app)
 
-![landingPage](https://user-images.githubusercontent.com/95094057/218668398-2e6bbe3c-c4d1-41c4-a1f3-184c94836ac4.png)
+![landingPage](https://user-images.githubusercontent.com/95094057/gitopener.vercel.app/public/images/readme/landingPage.png)
```

Comment:
just drag & drop the images into github editor and github will automatically upload those files and then reference those images. Currently, it's not loading the images, do this for all images

Reply:
Check now